cloudify_cli/commands/executions.py

```diff
@@ -281,20 +281,26 @@ def manager_start(workflow_id,
              short_help='Cancel a workflow execution [manager only]')
 @cfy.argument('execution-id')
 @cfy.options.force(help=helptexts.FORCE_CANCEL_EXECUTION)
+@cfy.options.kill()
 @cfy.options.verbose()
 @cfy.options.tenant_name(required=False, resource_name_for_help='execution')
 @cfy.assert_manager_active()
 @cfy.pass_client()
 @cfy.pass_logger
-def manager_cancel(execution_id, force, logger, client, tenant_name):
+def manager_cancel(execution_id, force, kill, logger, client, tenant_name):
     """Cancel a workflow's execution
 
     `EXECUTION_ID` is the ID of the execution to cancel.
     """
     utils.explicit_tenant_name_message(tenant_name, logger)
-    logger.info('{0}Cancelling execution {1}'.format(
-        'Force-' if force else '', execution_id))
-    client.executions.cancel(execution_id, force)
+    if kill:
+        message = 'Killing'
+    elif force:
+        message = 'Force-cancelling'
+    else:
+        message = 'Cancelling'
+    logger.info('{0} execution {1}'.format(message, execution_id))
+    client.executions.cancel(execution_id, force=force, kill=kill)
```

Comment:
This means that there's a change in the rest client, but you didn't send me a PR about it (and I don't see a branch on the topic).

Reply:
rest-client lives in common now! :P
https://github.com/cloudify-cosmo/cloudify-common/pull/44/files#diff-c1239f5a046a14346e86ca0029acfd70R217

Reply:
That's... embarrassing :)